specificatie/raadpleeg-persoon.yaml

```diff
@@ -0,0 +1,102 @@
+openapi: 3.0.3
+info:
+  title: Raadpleeg Persoon endpoint definitie
+  version: 2.0.0
+  contact: {}
+paths:
+  /personen/{persoonIdentificatie}:
+    get:
+      operationId: GetPersoon
+      summary: Raadpleeg een persoon
+      description: |
+        Raadpleeg een (overleden) persoon.
```

Comment:
de toevoeging tussen haakjes vind ik hier verwarrend. dit lijkt een beetje te suggereren dan dit endpoint vooral bedoeld is om overleden personen te raadplegen. @CathyDingemanse wat vind jij?

Reply:
Ik interpreteer woorden tussen haakjes altijd als delen van zinnen die weggelaten mogen worden bij het lezen.
Zelf zou ik hier echter kiezen voor 'Raadpleeg een al dan niet overleden persoon.' wat n.m.m. net even duidelijker is.